### PR TITLE
fix: Boolean/number export values in react

### PR DESCRIPTION
**Pull request by MoizMasud**
Closes https://github.com/IBM/carbon-ui-builder/issues/235

### CODE REVIEW

File: src/fragment-components/a-numberinput.tsx [hunk 1]
```diff
@@ -233,23 +233,24 @@ export const componentInfo: ComponentInfo = {
 			imports: ['NumberInput'],
 			code: ({ json }) => {
 				return `<NumberInput
+					id="${json.id}"
```
Comment (zvonimirfras) [inline src/fragment-components/a-numberinput.tsx:236]: Why is this needed? Should it be a string or a number?

**Review by zvonimirfras — CHANGES_REQUESTED** [review]

File: src/fragment-components/a-numberinput.tsx (hunk 1 above)
Comment (MoizMasud) [reply src/fragment-components/a-numberinput.tsx:236]: Its required prop of number input and it should be a string i believe. 
Screenshot 2023-03-10 at 12 00 12 PM

Screenshot 2023-03-10 at 12 01 12 PM

**Review by zvonimirfras — APPROVED** [review]
